apps/twitch-bot/internal/command/addcommandalias.go

```diff
@@ -6,41 +6,39 @@ import (
 	"strings"
 
 	"github.com/gempir/go-twitch-irc/v3"
-	"github.com/senchabot-dev/monorepo/apps/twitch-bot/client"
 	"github.com/senchabot-dev/monorepo/apps/twitch-bot/internal/command/helpers"
-	"github.com/senchabot-dev/monorepo/apps/twitch-bot/server"
 )
 
 const ADD_COMMAND_ALIAS_INFO = "For example: !acmda [command_name] [command_alias(es) separated by space]"
 
-func AddCommandAliasCommand(client *client.Clients, server *server.SenchabotAPIServer, message twitch.PrivateMessage, commandName string, params []string) {
-	if !helpers.CanExecuteCommand(context.Background(), server, message) {
+func (s *commands) AddCommandAliasCommand(message twitch.PrivateMessage, commandName string, params []string) {
+	if !helpers.CanExecuteCommand(context.Background(), s.service, message) {
 		return
 	}
 	command, aliasCommands, check := helpers.GetAliasCommandCreateParams(params)
 	if !check {
-		client.Twitch.Say(message.Channel, ADD_COMMAND_ALIAS_INFO)
+		s.client.Twitch.Say(message.Channel, ADD_COMMAND_ALIAS_INFO)
 		return
 	}
 	twitchChannelId := message.RoomID
 
 	if infoText, check := helpers.ValidateAliasCommandsLength(aliasCommands); !check {
-		client.Twitch.Say(message.Channel, message.User.DisplayName+", "+infoText)
+		s.client.Twitch.Say(message.Channel, message.User.DisplayName+", "+infoText)
 		return
 	}
 
-	infoText, err := server.CreateCommandAliases(context.Background(), command, aliasCommands, twitchChannelId, message.User.DisplayName)
+	infoText, err := s.service.DB.CreateCommandAliases(context.Background(), command, aliasCommands, twitchChannelId, message.User.DisplayName)
```

Comment:
[for follow up pr] let's not use `s.service.DB` object directly. It's an abstraction leak: `commands` struct should use `service` to do this. If we do it that way, we can safely change the `service.CreateCommandAliases` implementation without worrying about other places that use `DB` directly.